scenarioo-client/app/scenario/scenario.controller.ts

```diff
@@ -237,6 +257,17 @@ function ScenarioController($filter, $routeParams,
         return isUseCaseAdded;
     }
 
+    function isAddedScenario(useCaseDiffInfo) {
```

Comment:
you may use ES6, we have babel to transpile. Alternatively you can migrate the file to typescript.

Reply:
Migrated the functions to ES6. I didn't want to upgrade to typescript, because I'm not 100% sure what else would need to be changed.